index.html

```diff
@@ -1,13 +1,101 @@
+            <div class="logo"> <span class="nav_list_item"><a href="#" class="nav_link">Layout 1</a></span></div>
+            <input type="checkbox" id="check">
+            <div class="hanburger">
+                <label for="check">
+                    <i class="fa-solid fa-bars"></i>
+                </label>
+            </div>
+            <div class="hanburger2">
+                <label for="check">
+                    <i class="fa-solid fa-xmark"></i>
+                </label>
+            </div>
+            <ul class="nav_list" id="menu">
+                <li class="nav_list_item"><a href="#" class="nav_link">About</a></li>
+                <li class="nav_list_item"><a href="#" class="nav_link">Service</a></li>
+                <li class="nav_list_item"><a href="#" class="nav_link">Contact</a></li>
+            </ul>
+        </nav>
+        <div class="main">
+            <div class="section">
+                <div class="section1">
+                    <h1 class="m">Blog Post Title</h1>
+                    <h3>By <span class="ref_h">Text</span></h3>
+                    <hr class="m hr_col">
+                    <p class="s1_para m"><i class="fa-regular fa-clock"></i> Posted on August 24, 2013 at 9:30 PM </p>
+                    <hr class="m hr_col">
+                    <img class="b_img" src="https://placehold.co/900x300" alt="img1">
+                    <hr class="m hr_col">
+                    <h3 class="p_t_m t_para">Lorem, ipsum dolor sit amet consectetur adipisicing elit. Voluptas aliquam
+                        obcaecati quisquam pariatur officiis quos aut sed quod, libero, molestiae, minima assumenda
+                        error deserunt adipisci architecto fuga laboriosam velit nemo.</h3>
+                    <p class="p_t_m t_para">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ducimus, rerum.
+                        Esse, repellendus voluptatem molestias sint sit nam dicta temporibus fuga excepturi! Animi
+                        error, atque et iure id cum rem blanditiis!</p>
+                    <p class="p_t_m t_para">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ducimus, rerum.
+                        Esse, repellendus voluptatem molestias sint sit nam dicta temporibus fuga excepturi! Animi
+                        error, atque et iure id cum rem blanditiis!</p>
+                    <p class="p_t_m t_para">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ducimus, rerum.
+                        Esse, repellendus voluptatem molestias sint sit nam dicta temporibus fuga excepturi! Animi
+                        error, atque et iure id cum rem blanditiis!</p>
+                    <p class="p_t_m t_para">Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ducimus, rerum.
+                        Esse, repellendus voluptatem molestias sint sit nam dicta temporibus fuga excepturi! Animi
+                        error, atque et iure id cum rem blanditiis!</p>
```

Comment:
css naming convention is not good.
Read this
https://en.bem.info/methodology/naming-convention/